base/task.jl

```diff
@@ -366,10 +366,38 @@ function wait(t::Task)
 end
 
 # Wait multiple tasks
-waitany(tasks) = _wait_multiple(tasks)
-waitall(tasks; failfast=false) = _wait_multiple(tasks, true, failfast)
 
-function _wait_multiple(waiting_tasks, all=false, failfast=false)
+"""
+    waitany(tasks; throw=false) -> (done_tasks, remaining_tasks)
+
+Wait until at least one of the given tasks have been completed.
+
+If `throw` is `true`, the `CompositeException` will be thrown when one of the
+completed tasks completes with an exception.
+
+The return value consists of two task vectors. The first one consists of
```

Comment:
wording nit: use a consistent declarative tense here. for example:
```suggestion
If `throw` is `true`, throw `CompositeException` when one of the
completed tasks completes with an exception.

Return two task vectors: the first one containing the completed tasks,
the second containing uncompleted tasks.
```

Reply:
I've fixed the tense in docstrings of waitany and waitall.